regcomp.c

```diff
@@ -13147,7 +13145,7 @@ S_regtail_study(pTHX_ RExC_state_t *pRExC_state, regnode_offset p,
         DEBUG_PARSE_MSG("");
         regprop(RExC_rx, RExC_mysv, REGNODE_p(val), NULL, pRExC_state);
         Perl_re_printf( aTHX_
-                      "~ attach to %s (%" IVdf ") offset to %" IVdf "\n",
+                      "~ attach to %s (%zd) offset to %td\n",
                       SvPV_nolen_const(RExC_mysv),
                       (IV)val,
                       (IV)(val - scan)
```

Comment:
You left the `(IV)` casts in; do you need to remove them for `%zd`?

Reply:
Yes, I should have removed the casts; now done